docs/user/openstack/README.md

```diff
@@ -33,7 +37,47 @@ clouds:
+If you choose to use environment variables in place of a clouds.yaml, or along side it, the following variables are available:
+
+| Environment Variable |    Definition   | API Version |
+|----------------------|-----------------|:-----------:|
+| OS_CLOUD             | OpenStack cloud to use| |
+| OS_IDENTITY_API_VERSION | Version of identity api to use. Either v2 or v3| |
+| OS_AUTH_URL          | URL address of the OpenStack auth server | v2,v3 |
+| OS_AUTH_TOKEN        | Pre-generated X-Auth-Token that can be used in place of your username and password | v2, v3 |
+| OS_USERNAME          | Username is the username of the user | v2, v3 |
+| OS_USER_ID           | UserID is the unique ID of a user | v2, v3 |
+| OS_PASSWORD          | Password is the password of the user | v2, v3 |
+| OS_PROJECT_ID        | ID of the OpenStack project you want to deploy OCP in. It can be used to scope a user to a specific project | v2, v3 |
+| OS_PROJECT_NAME      | Name of the OpenStack project you want to deploy OCP in. ProjectName on its own is not enough to ensure a unique scope. It must also be combined with either a ProjectDomainName or ProjectDomainID. ProjectName cannot be combined with ProjectID in a scope | v2, v3 |
+| OS_DOMAIN_ID         | ID of a domain which can be used to identify the source domain of either a user or a project. If UserDomainID and ProjectDomainID are not specified, then DomainID is used as a default choice. It can also be used be used to specify a domain-only scope | v3 |
+| OS_DOMAIN_NAME       | Name of a domain which can be used to identify the source domain of either a user or a project. If UserDomainName and ProjectDomainName are not specified, then DomainName is used as a default choice. It can also be used be used to specify a domain-only scope | v3 |
+| OS_DEFAULT_DOMAIN    | Domain ID to fall back on if no other domain has been specified and a domain is required for scope | v3 |
+| OS_PROJECT_DOMAIN_ID | ID of the domain where a project resides. ProjectDomainID can be used in addition to a ProjectName when scoping a user to a specific project | v3 |
+| OS_PROJECT_DOMAIN_NAME | Name of the domain where a project resides. ProjectDomainName can be used in addition to a ProjectName when scoping a user to a specific project | v3 |
+| OS_USER_DOMAIN_ID    | ID of the domain where a user resides | v3 |
+| OS_USER_DOMAIN_NAME  | Name of the domain where a user resides | v3 |
+| OS_APPLICATION_CREDENTIAL_ID | Application Credential ID to login with | v3 |
+| OS_APPLICATION_CREDENTIAL_NAME | Application Credential name to login with | v3 |
+| OS_APPLICATION_CREDENTIAL_SECRET | Application Credential secret to login with | v3 |
+
+
+## OpenStack Requirements
+
+### Hardware Minimum
+
+With the latest version of the installer, we recommend the following minimum hardware in order to get a succesful deployment.
+
+   * 2 Available Floating IP Addresses
+   * Masters:
+     * 16 GB RAM each
```

Comment:
Since we're specifying the worker count, can you please add `Minimum 3` here too?

Also mention the bootstrap node (using the same flavour) and that it will be deleted post-deployment.

Something like:

```
* Bootstrap node:
  * 16 GB RAM
  * Used during the deployment
  * Will be deleted automatically after the cluster bootstrapping phase completes
```